jekyll/asset/script/main.js

```diff
@@ -234,43 +234,54 @@
     }
 
     function readingLevelOutputControl(element) {
-        $(document).on('lint-results', function (event, results, id) {
 
-            // trying to match the results to the counters
-            if (element.className.indexOf(id) < 0) {
-              return;
-            }
+      element.innerHTML = templates.readingLevel.render({"readingLevel" : 'N/A'});
+      cuff(element);
+        
+      $(document).on('lint-results', function (event, results, id) {
+        // trying to match the results to the counters
+        if (element.className.indexOf(id) < 0) {
+          return;
+        }
 
-            var tooHigh = results.readingLevel >= 9;
-            var readingLevelSummary = {
-              "readingLevel": results.readingLevel,
-              "tooHigh": tooHigh,
-              "level": tooHigh ? "error-highlight" : "info-highlight"
-            };
-            element.innerHTML = templates.readingLevel.render(readingLevelSummary);
-            cuff(element);
-            $(document).trigger('update-average', [id, results.readingLevel]);
-        });
+        var tooHigh = results.readingLevel >= 9;
+        var readingLevelSummary = {
+          "readingLevel": results.readingLevel < 0 ? 'N/A' : results.readingLevel,
+          "tooHigh": tooHigh,
+          "level": tooHigh ? "error-highlight" : "info-highlight"
+        };
+        element.innerHTML = templates.readingLevel.render(readingLevelSummary);
+        cuff(element);
+        $(document).trigger('update-average', [id, results.readingLevel]);
+      });
     }
 
     function averageRLOutputControl(element) {
 
-        var levels = {};
+      var levels = {};
+      element.innerHTML = templates.readingLevel.render({"readingLevel" : 'N/A'});
+      cuff(element);
 
-        $(document).on('update-average', function (event, levelId, readingLevel) {
+      $(document).on('update-average', function (event, levelId, readingLevel) {
```

Comment:
This might be better as a function call instead of an event. It'll be more direct and easier to debug that way.